ranch/ranch_test.go

```diff
@@ -199,7 +199,7 @@ func TestAcquire(t *testing.T) {
 
 	for _, tc := range testcases {
 		c := makeTestRanch(tc.resources)
-		res, err := c.Acquire(tc.rtype, tc.state, tc.dest, tc.owner, "")
+		res, _, err := c.Acquire(tc.rtype, tc.state, tc.dest, tc.owner, "")
```

Comment:
maybe check the `createdAt` value here to ensure it's set to something reasonable.

Reply:
I thought unsetting createdAt is fine.
I realize now that it would make the observations of those samples without requestID into infinite bucket.
The stats based on the metric would not be accurate if we have so many requests without IDs.

I am not sure what time sounds reasonable.

Maybe scrape only for the requests with IDs?